src/QSymbolicsBase/rules.jl

```diff
@@ -131,11 +82,58 @@ julia> qsimplify(anticommutator(σˣ, σˣ), rewriter=qsimplify_anticommutator)
 function qsimplify(s; rewriter=nothing)
     if QuantumSymbolics.isexpr(s)
         if isnothing(rewriter)
-            Fixpoint(Prewalk(Chain(RULES_ALL)))(s)
+            Fixpoint(Prewalk(Chain(RULES_SIMPLIFY)))(s)
         else
             Fixpoint(Prewalk(rewriter))(s)
         end
     else
         error("Object $(s) of type $(typeof(s)) is not an expression.")
     end
+end
+
+##
+# Expansion rules
+## 
+
+RULES_EXPAND = [
+    @rule(commutator(~o1, ~o2) => (~o1)*(~o2) - (~o2)*(~o1)),
```

Comment:
Are the tildas necessary on the right side of the mapping?

Reply:
Yes. See docs: https://docs.sciml.ai/SymbolicUtils/stable/manual/rewrite/.